gatekeeper/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "gatekeeper",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "gatekeeper",
+  "metric_prefix": "gatekeeper.",
+  "metric_to_check": "",
```

Comment:
```suggestion
  "metric_to_check": "gatekeeper.constraints",
```

Reply:
Not sure what we are trying to do here. Can you explain it, please? Thanks!

Reply:
yeah sorry! `metric_to_check` is used to automatically install the integration tile when the specified metric is detected/submitted. So if there are any dashboards/monitors/other assets associated with this integration, it will make them available to users.

Reply:
Cool! Thanks for the explanation! :today-i-learned: ;-)